PC/getpathp.c

```diff
@@ -245,7 +245,8 @@ static void
 join(wchar_t *buffer, const wchar_t *stuff)
 {
     if (_PathCchCombineEx_Initialized == 0) {
-        HMODULE pathapi = LoadLibraryW(L"api-ms-win-core-path-l1-1-0.dll");
+        HMODULE pathapi = LoadLibraryExW(L"api-ms-win-core-path-l1-1-0.dll", NULL,
```

Comment:
Here's what I was thinking:
```suggestion
         _PathCchCombineEx = NULL;
        if (GetProcAddress(GetModuleHandleW(L"kernel32.dll"), "AddDllDirectory") != NULL) {
            /* Check that we can use the LOAD_LIBRARY_SEARCH_SYSTEM32 flag below by ensuring
               the AddDllDirectory method exists first */
            HMODULE pathapi = LoadLibraryExW(L"api-ms-win-core-path-l1-1-0.dll", NULL,
                                            LOAD_LIBRARY_SEARCH_SYSTEM32);
            if (pathapi) {
                _PathCchCombineEx = (PPathCchCombineEx)GetProcAddress(pathapi, "PathCchCombineEx");
            }
        }

```

Reply:
(the suggestion isn't exactly correct since the GitHub suggestion was only for a single line)